sql-cli/sql_cli/run_dag.py

```diff
@@ -0,0 +1,161 @@
+from __future__ import annotations
```

Comment:
ATM, we have a module named `dag_generator.py`. What do you think about renaming it to `dag.py` and adding all the dag-related logic to it?